ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/sse/SSE.kt

```diff
@@ -58,14 +60,28 @@ public val SSE: ClientPlugin<SSEConfig> = createClientPlugin(
         )
     }
 
-    transformResponseBody { response, content, requestedType ->
-        if (content !is ClientSSESession) {
-            LOGGER.trace("Skipping non SSE response from ${response.request.url}: $content")
-            content
-        } else {
-            LOGGER.trace("Receive SSE session from ${response.request.url}: $content")
-            HttpResponseContainer(requestedType, content)
+    client.responsePipeline.intercept(HttpResponsePipeline.Transform) { (info, session) ->
```

Comment:
why not use `transformResponseBody` here?

Reply:
Because in `transformResponseBody` we have the next check: `if (content !is ByteReadChannel) return@intercept` and `ClientSSESession` isn't a `ByteReadChannel`

Reply:
ok